config/lingo.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    'uri'        => 'lingo',
+    'as'         => 'lingo.',
+    'middleware' => 'web',
+    'theme'      => env('LINGO_THEME','bulma'),
+];
```

Comment:
why do u need this ?

Reply:
add ability for user to change
- the theme from config file or .env (previously was MIX_LINGO_FRAMEWORK) 
- custom middleware